### PR TITLE
Update 1.4.2

### PR DESCRIPTION
**Pull request by oqyh**
Fix https://github.com/yonilerner/cs2-retakes-allocator/issues/106

Added `GiveNamedItem2` For Windows And Lunix Auto Create Sign at ../gamedata/RetakesAllocator.json

### CODE REVIEW

File: RetakesAllocatorCore/WeaponHelpers.cs [hunk 1]
```diff
@@ -48,6 +48,7 @@ public static class WeaponHelpers
 
     private static readonly ICollection<CsItem> _ctPistols = new HashSet<CsItem>
     {
+        CsItem.USP,
```
Comment (yonilerner) [inline RetakesAllocatorCore/WeaponHelpers.cs:51]:
```suggestion

```

**Review by yonilerner — APPROVED** [review]

**Comment by yonilerner** [comment]
thank you! you should be able to merge once ur ready

**Comment by yonilerner** [comment]
actually i still need to merge it.. ugh github sometimes is so annoying.

have you tested the build on this PR that it works? i can merge when youre ready, just @ me

**Comment by oqyh** [comment]
> actually i still need to merge it.. ugh github sometimes is so annoying.
> 
> have you tested the build on this PR that it works? i can merge when youre ready, just @ me

no problem 
good luck on your work <3

**Pull request merged**